distributed/client.py

```diff
@@ -4614,6 +4614,7 @@ def __init__(self, filename="dask-report.html"):
 
     async def __aenter__(self):
         self.start = time()
+        self.last = get_client().cluster.scheduler.monitor.count
```

Comment:
Can you think of a more descriptive name here ?  perhaps, `last_tick` or `last_count` ?

Reply:
Sure - `last_count` sounds good! Is it okay if I apply this change across `SystemMonitor`, since I got this name from there?

Reply:
I think that would be fine.  Thanks!